Add qfield docs (#808)
OSGeo/OSGeoLive-doc: doc/quickstart/qfield_quickstart.rst

Basic canvas interaction
========================

You can identify features by clicking on them.
In the itasca project, there are multiple layers identifiable, so you will get results from multiple layers, in this case here "airports" and "County Boundaries".

To inspect the attributes of one of the features, click on the feature in the identification panel.
You will see all attributes of this feature appear.

    .. image:: /images/projects/qfield/03_identify.png
      :scale: 70 %
      :alt: The hamburger menu

Legend
======

To see which layers are loaded and to inspect the layer legends, click the hamburger menu on the top left corner.

    .. image:: /images/projects/qfield/04_hamburger_menu.png
      :scale: 70 %
      :alt: The hamburger menu

You will see all layers in the project listed.
By right clicking on the layer (or long pressing on a touch screen) you will have additional actions
available for each layer.

    .. image:: /images/projects/qfield/05_legend.png
      :scale: 70 %
      :alt: The legend

Close the legend by clicking the white arrow on the top left or by clicking anywhere on the canvas
outside the legend panel.

Edit Data
=========

> By default, QField opens in read only mode and you are not able to make any changes to the data.

Reviewer: In read mode, features' attributes can be modified, it's their geometries that can't be modified.

```suggestion
By default, QField opens in read/view mode and you are not able to make any changes to preexisting geometries or add new features.
```

To go into digitize mode:

#. Click the hamburger menu on the top left corner, so you see the legend.
#. Click the pencil switch to go into digitize mode.

    .. image:: /images/projects/qfield/06_digitize_mode.png
      :scale: 70 %
      :alt: Switch to the digitize mode

Next, we will create a new airport.

> #. Activate the airports layer by clicking on the `airports` layer

Reviewer: ```suggestion
#. Activate the airports layer by clicking on the `airports` layer (note the current layer within which digitizing will occur is highlighted using a green background in the legend)
```

:Author: OSGeoLive
:License: Creative Commons Attribution-ShareAlike 3.0 Unported  (CC BY-SA 3.0)

@LOGO_qfield@
@OSGEO_KIND_qfield@
@VMDK_qfield@

********************************************************************************
@NAME_qfield@ Quickstart
********************************************************************************

QField is a powerful tool to collect data in the field for usage with QGIS.

.. contents:: Contents
   :local:

Start QField
============

#. Click the menu button at the bottom left
#. Click the QField icon in the Education menu

    .. image:: /images/projects/qfield/00_menu.png
      :scale: 70 %
      :alt: Find QField in the menu

You will see QField launch and show the changelog of the release.
Close the screen by clicking on the arrow button on the top left.

    .. image:: /images/projects/qfield/01_welcome_screen.png
      :scale: 70 %
      :alt: The changelog is shown once

Open a project
==============

Can we reuse an existing / pre-installed .qgs project? where from?

Navigation

> ==========

Reviewer 2: add new line after heading

Author: ```suggestion
==========

```
